### ISSUE TITLE
docker image shows unexpected version - 0.34 shows 0.32

### ISSUE THREAD

**Issue by yorickdowne** [ChainSafe/lodestar#3831]
**Describe the bug**

```
docker run chainsafe/lodestar:v0.34.0 --version
🌟 Lodestar: TypeScript Implementation of the Ethereum 2.0 Beacon Chain.
  * Version: v0.32.0/master/+81/b3842497 (git)
  * by ChainSafe Systems, 2018-2021
```

**Comment by wemeetagain** [comment]
Thanks @yorickdowne 
Should be fixed when we release v0.35.0

**Comment by yorickdowne** [comment]
Confirmed that this has been fixed. Thank you!
